.github/workflows/nodejs.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: yarn install, build, and test
         run: |
           yarn
-          yarn check
+          yarn run check
```

Comment:
:mega: I believe you need to run `yarn postinstall` beforehand for this to pass. Without the files inside `gen/` it won't compile.

Reply:
`yarn` does `yarn install`.
The `postinstall` script is always ran automatically after `install`.

Reply:
Fair enough, thanks for the explanation! 

I do have to say though that I hate that it's implicit.

Reply:
well, I think it's a npm standard here to allow you to do exactly that.